src/ripple/basics/hardened_hash.h

```diff
@@ -59,12 +59,12 @@ make_seed_pair() noexcept
     {
         std::mutex mutex;
         std::random_device rng;
-        std::mt19937_64 gen {rng()};
+        std::mt19937_64 gen;
         std::uniform_int_distribution <std::uint64_t> dist;
 
         state_t() : gen(rng()) {}
-        // state_t(state_t const&) = delete;
-        // state_t& operator=(state_t const&) = delete;
+        state_t(state_t const&) = delete;
+        state_t& operator=(state_t const&) = delete;
```

Comment:
The main reason I hesitate is this link:
http://blogs.msdn.com/b/vcblog/archive/2014/06/11/c-11-14-feature-tables-for-visual-studio-14-ctp1.aspx
shows "partial" support for deleted functions in VS2013.  I dunno which parts are supported and which are not.  But I agree with you.  Visual Studio can probably chew through these.